darwin/Platform.c

```diff
@@ -18,7 +18,11 @@ in the source distribution for its full text.
 #include <net/if_types.h>
 #include <net/route.h>
 #include <sys/socket.h>
+#if MAC_OS_X_VERSION_MIN_REQUIRED < 1090
+#include <mach/mach_port.h>
+#else
 #include <sys/_types/_mach_port_t.h>
+#endif
```

Comment:
This should go into the list further down with the rest of conditional defines.

Also compare re `HAVE_MACH_MACH_TIME_H` defined via `configure.ac`/`config.h`.

If checking this via the availability macros, a check could be placed like this:
```suggestion
#include <AvailabilityMacros.h>
#if MAC_OS_X_VERSION_MIN_REQUIRED < 1090
#include <mach/mach_port.h>
#else
#include <sys/_types/_mach_port_t.h>
#endif
```

or

```suggestion
#if HAVE_MACH_MACH_PORT_H
#include <mach/mach_port.h>
#else
#include <sys/_types/_mach_port_t.h>
#endif
```